presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -451,7 +453,7 @@ void PrestoServer::run() {
             exchangeHttpCpuExecutor_.get(),
             exchangeHttpIoExecutor_.get(),
             exchangeSourceConnectionPool_.get(),
-            sslContext_);
+            exchangeSslContext_);
```

Comment:
Instead of separting it, better we use same context for everything and use either http2 or 1 for all http communication. But lets hear from @NikhilCollooru first how reactor.netty-http-client-enabled is working with Prestissimo workers.